ddtrace/contrib/internal/pylibmc/client.py

```diff
@@ -41,7 +41,13 @@ def __init__(self, client=None, service=memcached.SERVICE, tracer=None, *args, *
         if not isinstance(client, _Client):
             # We are in the patched situation, just pass down all arguments to the pylibmc.Client
             # Note that, in that case, client isn't a real client (just the first argument)
-            client = _Client(client, *args, **kwargs)
+            if kwargs.get("servers"):
```

Comment:
```suggestion
            if "servers" in kwargs:
```

We probably don't care about the truthiness of `servers` ?, e.g. if they did `servers=[]` (not sure why, but hey, it is possible I guess)